data/json/furniture.json

```diff
@@ -3,6 +3,7 @@
         "type" : "furniture",
         "id" : "f_hay",
         "name": "hay",
+        "description": "A bay of hay. Feed it to your horse, or sleep on it.",
```

Comment:
There are no horses that can be fed with hay yet. The descriptions shouldn't mention features that don't exist.

Reply:
Also, again: "bale" not "bay"